source/topics/paas/how-to-order-certs.md

```diff
@@ -0,0 +1,19 @@
+This error is the result of either not including the necessary intermediate certs, or due to improper formatting.
```

Comment:
The content of this article is very strong. Minor copy edit for syntax: remove "due to".